test/gurka/test_gtfs.cc

```diff
@@ -136,6 +129,13 @@ TEST(GtfsExample, WriteGtfs) {
     .wheelchair_boarding = "1",
   };
   feed.add_stop(nemo_platform);
+  struct gtfs::Stop nemo {
+    .stop_id = stopOneID, .stop_name = gtfs::Text("POINT NEMO"), .coordinates_present = true,
+    .stop_lat = station_one_ll->second.second, .stop_lon = station_one_ll->second.first,
+    .parent_station = "", .location_type = gtfs::StopLocationType::Station,
+    .stop_timezone = "America/Toronto", .wheelchair_boarding = "1",
+  };
+  feed.add_stop(nemo);
```

Comment:
now we're adding the stations for nemo last, after the platform & egress. before this PR, this failed.